src/directives/form-errors.directive.spec.ts

```diff
@@ -351,6 +356,71 @@ describe("NgxFormErrorsDirective", () => {
 				expect(mockObserver.complete).not.toHaveBeenCalled(); // the changes observable should never complete!
 			});
 
+			it("should contain the message defined via the NgxFormErrorsMessageService for that specific validation under that specific formControlName or just the validation name if no message defined", () => {
```

Comment:
I don't think we should have a separate `it` for this tests... because it feels like we are cheating by setting only the message we expect (`formControlName.error`). So I suggest to merge this into the current `it` we have so we can really see that we cover both cases properly ;)